workload_7_output.awk

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/awk -f
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+BEGIN {
+  IS_DATA="^202.*";
```

Comment:
optional: maybe a better regex would be `^202[0-9]-\d{2}-\d{2}`, but I forget if `awk` supports `\d` and `{2}`, your call, don't make your life more difficult just for this.